e2e/client/playwright/page-object-models/settings/content-profile.ts

```diff
@@ -42,4 +42,31 @@ export class ContentProfileSettings {
         // wait for saving to finish and modal to close
         await expect(this.page.locator(s('content-profile-edit-view'))).not.toBeVisible();
     }
+
+    async addFieldToContentProfile(
```

Comment:
since we changed it to accept multiple fields - the name should be in plural too - `addFieldToContentProfile`